content/logo/past.mdx

```diff
@@ -0,0 +1,51 @@
+---
+seo:
+  title: SSW Logo Index
+  canonical: ''
+  showBreadcrumb: true
+header: SSW Past Logos Index
+_body:
+  - title: Our Past logos
+    grids:
+      - gridTitle: Our Past logos
+        blocks:
+          - title: Old AI HackDay
+            image: /images/company-logos/zzAIHackday_preview.png
+          - title: Old Xamarin HackDay
+            image: /images/company-logos/XamarinHackDay_preview.png
+          - title: Old Angular HackDay
+            image: /images/company-logos/zzAngularHackDay_preview.png
+          - title: SSW 1994-2014
+            image: /images/company-logos/SSWLogo_preview.png
+          - title: SSW TV 2009-2021
+            image: /images/company-logos/SSWTV_preview.png
+          - title: Old SugarLearning
+            image: /images/company-logos/zzSugarLearning_preview.png
+          - title: Old SmashingBarrier
+            image: /images/company-logos/zzSmashingBarrier_preview.png
+          - title: Old LinkAuditor
+            image: /images/company-logos/zzLinkAuditor_preview.png
+          - title: Old CodeAuditor
+            image: /images/company-logos/zzCodeAuditor_preview.png
+          - title: Old HealthCheck
+            image: /images/company-logos/HealthCheck_preview.png
+          - title: Old DataOnion
+            image: /images/company-logos/DataOnion_preview.png
+          - title: Old SSRSAuditor
+            image: /images/company-logos/SSRSAuditor_preview.png
+          - title: OLD SQLDeploy
+            image: /images/company-logos/SQLDeploy-2010_preview.png
+          - title: Old TimePRO
+            image: /images/company-logos/TimePRO-old_preview.png
+          - title: Old .NET User Group
+            image: /images/company-logos/old-NETUGSYD_logo_preview.png
+          - {}
```

Comment:
```suggestion
```

Reply:
im using an empty object here to insert a blank spot in the grid. The 3 prev blocks require 2 images, hence an empty block followed by 3 images with no title

Reply:
On other viewports it looks a bit odd:

![image](https://github.com/SSWConsulting/SSW.Website/assets/20507092/8352d61c-5775-4bb1-a43e-aad85e83352a)

Reply:
Fixed this by allowing a second 'related' image to each block removing the need for an empty block and also fixing the weird responsive behaviour.